syntax = "proto3";

message CreateDeck {
  string name = 1;
}
